app/tests/beaker-plugin-safe-app/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I think we should have these dev dependencies in the `app/package.json` so they're installed automatically for developers.

We can then add a `scripts` section to this and a script to run the SAFE API test server, for eg.

This'll remove the extra yarn/package files and keep everything in one place.